src/resources/input.csv

```diff
@@ -0,0 +1,8 @@
+b,banana,20
```

Comment:
bad location
should be `src/main/resources`
idea will give this folder a special icon